sections/appendix/process.tex

```diff
@@ -0,0 +1,30 @@
+\subsection{Project Process}
+\begin{itemize}
+	\item Initial Idea - Combine LDA and PageRank (clustering)
+	\item Preprocessing \& LDA implementation
+	\item Adjacency matrix construction
+	\begin{itemize}
+		\item challenges with calculation time
+		\item focus on more restricting preprocessing and thresholds
+	\end{itemize}
+	\item Bad results
+	\item Discovery of related papers
+	\item Found out our model didn't converge
+\end{itemize}
+
+Our initial idea with the project was to combine \gls{lda} with a PageRank algorithm to see if this could improve the information retrival results of \gls{lda}.
```

Comment:
```suggestion
Our initial idea with the project was to combine \gls{lda} with a PageRank algorithm to see if this could improve the information retrieval results of \gls{lda}.
```